configuration/pih/petl/jobs/sql/extractions/mch_kellogg_data.sql

```diff
@@ -0,0 +1,333 @@
+-- set @locale = 'en';  -- uncomment for testing
+
+set sql_safe_updates = 0;
+set session group_concat_max_len = 100000;
+
```

Comment:
what is the purpose of the lines above?

Reply:
I removed them.  If safe updates isn't turned off, it doesn't let you update a table without a where clause, which we don't need in this script.

And in some of our scripts (mostly exports, not ETLs) we have columns that exceed the default group_concat limit.  But again, don't need it for this script.